src/modules/InfiniteHubStories/InfiniteHubStories.module.scss

```diff
@@ -0,0 +1,88 @@
+.newsrooms {
+    display: grid;
+    gap: $spacing-6;
+    grid-template-columns: repeat(5, 1fr);
+    margin-bottom: $spacing-10;
+
+    @include desktop-up {
+        &[data-count="1"] {
+            grid-template-columns: 33.33%;
+            justify-content: center;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        &[data-count="2"] {
+            grid-template-columns: repeat(2, 33.33%);
+            justify-content: center;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        &[data-count="3"],
+        &[data-count="5"],
+        &[data-count="6"],
+        &[data-count="9"] {
+            grid-template-columns: repeat(3, 1fr);
+        }
+
+        &[data-count="4"],
+        &[data-count="7"],
+        &[data-count="8"] {
+            grid-template-columns: repeat(4, 1fr);
+        }
+    }
+
+    @include tablet-only {
+        grid-template-columns: repeat(3, 1fr);
+
+        &[data-count="1"] {
+            grid-template-columns: 373.33px;
```

Comment:
🤔  very precise. where does this number come from?

Reply:
It's 1120px / 3. I'm not sure why @iamhugofelix used this container width because our containers are 1200px wide, so maybe we can consider using 1200/3 which is 400px. What do you think @iamhugofelix ?

Reply:
Now that I think about it, I guess your initial width makes sense as it's one column out of three, centered, right?

So it's 1200-2*40 (gap) / 3.

So let's keep it, I'll just add the calculation in the CSS instead of having the explicit number there.